src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -169,6 +169,184 @@
  * </li>
  * </ul>
  * <p>
+ *     To configure the check :
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;CustomImportOrder&quot;/&gt;
+ * </pre>
+ * <p>
+ * Example:-
+ * </p>
+ * <pre>
+ * package com.company;
+ * import org.apache.commons.io.fileutils; //OK
```

Comment:
Change `fileutils` to any other name. It is not in the dictionary so it giving a spell check error.